### PR TITLE
RHDEVDOCS-4319 - Clarify scope of infra

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise- 4.8, 4.9, 4.10, 4.11

JIRA issue:
https://issues.redhat.com/browse/RHDEVDOCS-4319

Preview pages: https://drive.google.com/file/d/1VeKLMue4Xqf1X8ie8ldtKYGaUPDf7iXo/view?usp=sharing

SME+QE review: @saumeya 
Peer-review: @rolfedh

### CODE REVIEW

File: modules/go-add-infra-nodes.adoc [hunk 1]
```diff
@@ -20,7 +34,7 @@ $ oc label node <node-name> node-role.kubernetes.io/infra=
 ----
 $ oc edit gitopsservice -n openshift-gitops
 ----
-. In the `GitOpsService` CR file, add `runOnInfra` toggle to the `spec` section and set it to `true`:
+. In the `GitOpsService` CR file, add `runOnInfra` toggle to the `spec` section and set it to `true`. This toggle will move the workloads from `openshift-gitops` namespace to the infrastructure nodes:
```
Comment (saumeya) [inline modules/go-add-infra-nodes.adoc:37]: workloads *in (not from) `openshift-gitops`

File: modules/go-add-infra-nodes.adoc [hunk 2]
```diff
@@ -4,7 +4,21 @@
 
 :_content-type: PROCEDURE
 [id="add-infra-nodes_{context}"]
-= Adding infrastructure nodes
+= Moving workloads into infrastructure nodes
```
Comment (saumeya) [inline modules/go-add-infra-nodes.adoc:7]: *to infrastructure nodes

**Comment by saumeya** [comment]
LGTM, Thanks @DebarghoGhosh

**Review by saumeya — APPROVED** [review]

File: cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc [hunk 3]
```diff
@@ -6,9 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can use infrastructure nodes to isolate infrastructure workloads for two primary purposes:
+Infrastructure nodes prevent additional billing cost against subscription counts.
```
Comment (rolfedh) [inline cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc:9]: I recommend preserving the user-centered tone and explanation in the original phrase even though the detail have changed.

File: cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc [hunk 4]
```diff
@@ -6,9 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can use infrastructure nodes to isolate infrastructure workloads for two primary purposes:
+Infrastructure nodes prevent additional billing cost against subscription counts. 
 
-* Prevent incurring billing costs against subscription counts.
-* Separate maintenance and management.
+{product-title} allows certain workloads installed by the {gitops-title} Operator to run on Infrastructure nodes. This comprises the workloads that are installed by the {gitops-title} Operator by default in the `openshift-gitops` namespace, including the default Argo CD instance in that namespace.
```
Comment (rolfedh) [inline cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc:11]: IBM SG: [Allow:] Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

Comment (rolfedh) [reply cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc:11]: Most instances of "infrastructure node" in openshift-docs are not capitalized. (...also "worker node" and "compute node")

File: modules/go-add-infra-nodes.adoc [hunk 5]
```diff
@@ -20,7 +34,7 @@ $ oc label node <node-name> node-role.kubernetes.io/infra=
 ----
 $ oc edit gitopsservice -n openshift-gitops
 ----
-. In the `GitOpsService` CR file, add `runOnInfra` toggle to the `spec` section and set it to `true`:
+. In the `GitOpsService` CR file, add `runOnInfra` toggle to the `spec` section and set it to `true`. This toggle will move the workloads in `openshift-gitops` namespace to the infrastructure nodes:
```
Comment (rolfedh) [inline modules/go-add-infra-nodes.adoc:37]:
```suggestion
. In the `GitOpsService` CR file, add `runOnInfra` field to the `spec` section and set it to `true`. This field moves the workloads in `openshift-gitops` namespace to the infrastructure nodes:
```
- "Toggle" is non-standard terminology. Please replace it with established terminology. 
- Avoid using future tense, "will."

File: cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc (hunk 3 above)
Comment (rolfedh) [reply cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc:9]:
```suggestion
Infrastructure nodes prevent additional billing costs against subscription counts. 
```

File: cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc (hunk 4 above)
Comment (rolfedh) [reply cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc:11]: Suggestion: "specific workloads".

Comment (rolfedh) [reply cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc:11]: Suggestion (active voice): "These include workloads that the {gitops-title} Operator installs by default in the openshift-gitops namespace, including the default Argo CD instance in that namespace."

File: modules/go-add-infra-nodes.adoc [hunk 6]
```diff
@@ -4,7 +4,21 @@
 
 :_content-type: PROCEDURE
 [id="add-infra-nodes_{context}"]
-= Adding infrastructure nodes
+= Moving workloads to infrastructure nodes
+
+The default workloads that can be moved to the infrastructure nodes are:
```
Comment (rolfedh) [inline modules/go-add-infra-nodes.adoc:9]: This title and abstract seem incomplete. How about a tldr like this one:

```suggestion
= Moving GitOps infrastructure workloads to infrastructure nodes

To running GitOps control plane workloads on infrastructure nodes, you label existing nodes as infrastructure and move the workloads in the `openshift-gitops` namespace to them.

You can move the following default workloads to infrastructure nodes:
```

Comment (DebarghoGhosh) [reply modules/go-add-infra-nodes.adoc:9]: @rolfedh changing it to Moving GitOps workloads to infrastructure nodes

File: modules/go-add-infra-nodes.adoc [hunk 7]
```diff
@@ -4,7 +4,21 @@
 
 :_content-type: PROCEDURE
 [id="add-infra-nodes_{context}"]
-= Adding infrastructure nodes
+= Moving GitOps workloads to infrastructure nodes
+
```
Comment (rolfedh) [inline modules/go-add-infra-nodes.adoc:8]: As suggested earlier, every module must contain an abstract that summarizes its content in a brief user-centered statement.

**Review by rolfedh — CHANGES_REQUESTED** [review]

**Review by rolfedh — APPROVED** [review]

**Comment by JStickler** [comment]
Received confirmation on Slack that this should go into 4.12 as well.

**Pull request merged**